### PR TITLE
dlgOneWayANOVA

### PR DESCRIPTION
**Pull request by nyagishira**
the dialog is not complete, am still working on it although i would appreciate comments

### CODE REVIEW

File: instat/dlgOneWayAnova.vb [hunk 1]
```diff
@@ -1,62 +1,82 @@
...
+
+    End Sub
+
+    Private Sub InitialiseDialog()
         ucrBase.clsRsyntax.SetFunction("aov")
         ucrBase.clsRsyntax.iCallType = 2
         ucrYVariate.Selector = ucrAddRemoveDataFrame
-        ucrYVariate.SetMeAsReceiver()
         ucrFactor.Selector = ucrAddRemoveDataFrame
-        ucrFactor.SetDataType("factor")
```
Comment (dannyparsons) [inline instat/dlgOneWayAnova.vb]: I think this line is still needed in `InitialiseDialog()`?

File: instat/dlgOneWayAnova.vb [hunk 2]
```diff
@@ -1,62 +1,82 @@
...
 
-        If ((Not (strYVariate = "")) And (Not (strFactor = ""))) Then
-            ucrBase.clsRsyntax.AddParameter("formula", strYVariate & "~" & strFactor)
+    Private Sub ucrYVariate_SelectionChanged(sender As Object, e As EventArgs) Handles ucrYVariate.SelectionChanged
+        clsModel.SetParameter(True, strValue:=ucrYVariate.GetVariableNames(bWithQuotes:=False))
+        TestOKEnabled()
+    End Sub
 
+    Private Sub ucrFactor_SelectionChanged(sender As Object, e As EventArgs) Handles ucrFactor.SelectionChanged
+        clsModel.SetParameter(False, strValue:=ucrYVariate.GetVariableNames(bWithQuotes:=False))
```
Comment (dannyparsons) [inline instat/dlgOneWayAnova.vb]: You have `ucrYVariate `here when I think you want `ucrFactor`.

**Comment by nyagishira** [comment]
I have made some corections

**Comment by dannyparsons** [comment]
This is looking really good, keep going! Let me know when you want it reviewed again.

**Pull request merged**